src/components/ui/primary-button.tsx

```
@@ -0,0 +1,61 @@
+
+export interface PrimaryButtonProps extends ButtonHTMLAttributes<HTMLButtonElement> {
+  isLoading?: boolean
+}
+
+const PrimaryButton = forwardRef<HTMLButtonElement, PrimaryButtonProps>(
+  ({ className, children, isLoading, disabled, ...props }, ref) => {
+    return (
+      <button
+        ref={ref}
+        className={cn(
+          'w-full px-3 py-2 text-sm font-semibold text-white bg-blue-600 rounded-md',
```

ESLint

[error] Replace `px-3·py-2·text-sm·font-semibold·text-white·bg-blue-600·rounded-md` with `rounded-md·bg-blue-600·px-3·py-2·text-sm·font-semibold·text-white`

(prettier/prettier)